tests/integration/features.go

```diff
@@ -465,7 +465,7 @@ func GetAllLogsFeature(waitFunction stepfuncs.WaitForLogs, generate bool) featur
 		feature = feature.
 			Teardown(removeLogsDeployment).
 			Teardown(removeLogsDaemonset).
-			Teardown(stepfuncs.KubectlDeleteNamespaceOpt(internal.LogsGeneratorNamespace))
+			Teardown(stepfuncs.KubectlDeleteNamespaceOpt(internal.LogsGeneratorNamespace, true))
```

Comment:
Is the primary reason for this change to speed up namespace termination?

Reply:
Yes.